### PR TITLE
Fix ordered map warnings

### PR DESCRIPTION
**Pull request by alxvth**
Just some minor warnings.

### CODE REVIEW

File: ManiVault/src/plugins/TextData/src/OrderedMap.cpp [hunk 1]
```diff
@@ -77,7 +77,7 @@ namespace
         loadFromDisk(variantMap, combinedList);
 
         // Iterate over the big stringlist and extract the key-value pairs
-        for (int i = 0; i < combinedList.size(); )
+        for (size_t i = 0; i < static_cast<size_t>(combinedList.size()); )
```
Comment (JulianThijssen) [inline ManiVault/src/plugins/TextData/src/OrderedMap.cpp:80]: As .size() returns a sizetype_t would it not be cleaner to make the type of i sizetype_t?

File: ManiVault/src/plugins/TextData/src/OrderedMap.cpp [hunk 2]
```diff
@@ -128,6 +128,7 @@ namespace mv
         catch (std::out_of_range& e)
         {
             qWarning() << "Failed to find column: " << columnName;
+            qWarning() << "Caught exception: " << e.what();
```
Comment (JulianThijssen) [inline ManiVault/src/plugins/TextData/src/OrderedMap.cpp:131]: Is this necessary? The out of range exception gets called if it wasn't able to find the key in the map. I don't think more information is helpful. Perhaps slightly more elaboration in the original message.

**Comment by JulianThijssen** [comment]
Thanks for fixing it!

**Review by JulianThijssen — CHANGES_REQUESTED** [review]

File: ManiVault/src/plugins/TextData/src/OrderedMap.cpp (hunk 2 above)
Comment (alxvth) [reply ManiVault/src/plugins/TextData/src/OrderedMap.cpp]: No, I mainly wanted to get rid of the warning. Marking it as [[maybe_unused]] gets the job done as well.

Comment (JulianThijssen) [reply ManiVault/src/plugins/TextData/src/OrderedMap.cpp]: Ah I see, should be able to remove the variable name as well right? just catch (std::out_of_range) or something

Comment (JulianThijssen) [reply ManiVault/src/plugins/TextData/src/OrderedMap.cpp]: But both are fine

**Pull request merged**